setup.cfg

```diff
@@ -25,6 +25,6 @@ install_requires =
 setup_requires =
     setuptools
 packages = rsconnect_jupyter
-python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+python_requires = !=2.*, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
```

Comment:
This matches rsconnect-python so is OK. Could also consider making this `>=3.5`; the more complex pattern was needed previously to give us a discontiguous range.